news/2023/2023-11-08-OWC2023-RO16-recap.md

```diff
@@ -0,0 +1,53 @@
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/SlWKKA-ltZY" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Without further ado, let's look at some highlighted matches from the Round of 16!
+
+### ::{ flag=DE }:: Germany vs. ::{ flag=CA }:: Canada
+
+Yes, you are reading this correctly.
+
+Due to Canada's unusual 11th seed placing ([due to some players playing qualifiers at TwitchCon with technical issues](https://twitter.com/xootynator/status/1716179172270166224)), this battle of tournament veterans happened in RO16. The spectators were expecting a coin flip match, well represented by the [osu! World Cup pick'ems](https://pickem.hwc.hr/tournaments/130)[^1]. The North American squad had the better start going up 3-1 including an important break point on [Mysteka - Jerrybeans](https://osu.ppy.sh/beatmapsets/1860553#osu/3825151). Germany responded quickly on the low approach rate map [jun with Alison - SUNKiSS DROP ~jun side~](https://osu.ppy.sh/beatmapsets/1498309#osu/3070802), claiming the break point right back. [Criller](https://osu.ppy.sh/users/8116659)'s double S and [Shiox](https://osu.ppy.sh/users/11921197)'s 1x100 score provided much needed confidence.
+
+However, the slug fest of break points continued on a highly contested [Yamamoto Mineko - Presea](https://osu.ppy.sh/beatmapsets/2082024#osu/4360325) pick. Eventually, this turned out to be the crucial point for Canada, and they converted their first match point into the seed upset 5-2 victory against the bronze medallist of 2022.
+
+[^1]: Scroll to the bottom of the page, select `Round of 16` and find the match marked `X`.
+
+### ::{ flag=TH }:: Thailand vs. ::{ flag=NZ }:: New Zealand
+
+The team from Thailand has been swept against the United States in RO32, while New Zealand just barely missed out on advancing in winners bracket against Italy. Thus, the confidence level from the Kiwis seemed quite high, and the odds were in their favour.
+
+As it happens, Thailand struggled quite a bit to win their own first two picks by a margin of less than 30k points on each map. However, New Zealand didn't manage to win a single break point while confidently winning their own picks which consequently led to the tiebreaker. Here, disaster struck for Thailand as [Deppyforce](https://osu.ppy.sh/users/5286213) disconnected during the last map and a restart was not possible anymore. New Zealand took the lead back and increased it — until Deppyforce resurrected and proceeded to full combo the tiebreaker along with [chests](https://osu.ppy.sh/users/14806365) and [Zoomer](https://osu.ppy.sh/users/6600930).
+
+The Kiwis have been eliminated after two heartbreaking tiebreaker losses whereas Thailand's roster will continue their journey thanks to their huge carry: [Lesperry](https://osu.ppy.sh/users/18092331).
```

Comment:
I will ask junihuhn to comment

Reply:
Ah good point. I really wanted to insert Lesperry somehow because they are the main carry of Thailand. We don't have to commit though. 

Alternative: "The Kiwis have been eliminated after two heartbreaking tiebreaker losses whereas Thailand's roster will continue their journey in Quarterfinals to face Ukraine next."